embedly/embedly.py

```diff
@@ -54,13 +52,21 @@ def get_urls_from_embedly(urls):
     except requests.RequestException:
         response = None
 
+    embedly_data = []
     if response is not None:
         try:
-            response_data = json.loads(response.content)
+            embedly_data = json.loads(response.content)
         except (TypeError, ValueError):
             pass
```

Comment:
Ignore exceptions should be fine for now. But once we have support for logging, we could log those exceptions for further investigation.